### PR TITLE
Master Link File

### LINKED ISSUE #1541: Move links from image3 to _links.md

### PR DESCRIPTION
**Pull request by jpstroop**
Will close #1541 when complete

### CODE REVIEW

File: source/_includes/links.md [hunk 1]
```diff
@@ -1,79 +1,131 @@
...
+[org-w3c-json-ld-scoped-contexts]: https://json-ld.org/spec/latest/json-ld/#scoped-contexts "JSON-LD Scoped Contexts"
+[org-w3c-json-ld]: http://www.w3.org/TR/json-ld/ "JSON-LD 1.0"
+[org-w3c-media-frags]: http://www.w3.org/TR/media-frags/ "Media Fragments"
+[org-w3c-rdf11-blank-nodes]: http://www.w3.org/TR/rdf11-concepts/#section-blank-nodes "RDF 1.1 Concepts"
+[org-w3c-svg]: http://www.w3.org/TR/SVG/ "Scalabe Vector Graphics"
+[org-w3c-webanno-protocol]: http://w3.org/TR/annotation-protocol/ "Web Annotation Protocol"
+[org-w3c-webanno]: http://w3.org/TR/annotation-model/ "Web Annotaiton Model"
+[org-w3c-webarch]: http://www.w3.org/TR/webarch/ "Architecture of the World Wide Web"
+[org-w3c-xsd-datetime]: https://www.w3.org/TR/xmlschema11-2/#dateTime "XSD DateTime"
+[orgs-w3c-json-ld]: http://www.w3.org/TR/json-ld/ "JSON-LD"
```
Comment (azaroth42) [inline source/_includes/links.md]: org-w3c-json-ld no?

File: source/api/annex/registry/behaviors/index.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ Please send feedback to [iiif-discuss@googlegroups.com][iiif-discuss]
 
 ## 1. Introduction
 
-This is one of a number of [IIIF registries][registry-home]. It lists a set of behaviors, previously called viewing hints, in the Presentation API that have been identified as useful for implementations.  They may be defined by the IIIF community, or outside of it.
+This is one of a number of [IIIF registries][registry]. It lists a set of behaviors, previously called viewing hints, in the Presentation API that have been identified as useful for implementations.  They may be defined by the IIIF community, or outside of it.
```
Comment (azaroth42) [inline source/api/annex/registry/behaviors/index.md]: Should registry have a prefix?

File: source/api/presentation/2.1/index.md [hunk 3]
```diff
@@ -697,7 +697,7 @@ Recommended URI pattern:
...
 
-Annotations _MAY_ have their own URIs, conveyed by adding an `@id` property to the JSON object, and if so _SHOULD_ be HTTP URIs. The content of the annotation _SHOULD_ be returned if the URI is dereferenced. Annotations _MAY_ be dereferenced separately from their annotation lists, sequences and manifests; some systems may do this and identifiers should be given using the recommended pattern if possible.
+Annotations _MAY_ have their own URIs, conveyed by adding .stable. property to the JSON object, and if so _SHOULD_ be HTTP URIs. The content of the annotation _SHOULD_ be returned if the URI is dereferenced. Annotations _MAY_ be dereferenced separately from their annotation lists, sequences and manifests; some systems may do this and identifiers should be given using the recommended pattern if possible.
```
Comment (zimeon) [inline source/api/presentation/2.1/index.md]: Bad search and replace? `.stable.`

File: source/api/image/3.0/index.md [hunk 4]
```diff
@@ -789,27 +789,27 @@ The order in which servers parse requests and detect errors is not specified. A
...
 
 ##  8. Authentication
 
 Images are generally secondary resources in a web page or application. In the case of web pages, images are embedded in the HTML `img` tag, and are retrieved via additional HTTP requests. When a user cannot load a web page, it is possible — and a generally accepted behavior — to redirect the user to another page and offer the opportunity to authenticate. This is not an option for secondary resources such as images, and the user is instead simply presented with a broken image icon.
 
-No new authentication mechanisms are proposed, nor roles for authorization business logic. Instead, it is expected that authentication requirements and processes are handled outside of any IIIF-specific context, but within a IIIF-aware access control workflow. Please see the draft [authentication][authentication-ext] specification.
+No new authentication mechanisms are proposed, nor roles for authorization business logic. Instead, it is expected that authentication requirements and processes are handled outside of any IIIF-specific context, but within a IIIF-aware access control workflow. Please see the draft [image3-authentication][iiif-auth] specification.
```
Comment (azaroth42) [inline source/api/image/3.0/index.md]: Bad image3- in the link text

File: source/api/presentation/2.1/index.md (hunk 3 above)
Comment (azaroth42) [inline source/api/presentation/2.1/index.md]: Bad replace for .stable.

File: source/api/presentation/3.0/index.md [hunk 5]
```diff
@@ -853,11 +853,11 @@ The value _MUST_ be an array of JSON objects. Each item _MUST_ have at least the
...
 
 While any resource _MAY_ be the `target` of an Annotation, this specification defines only motivations for Annotations that target Canvases. These motivations allow clients to determine how the Annotation should be rendered, by distinguishing between Annotations that provide the content of the Canvas, from ones with externally defined motivations which are typically comments about the Canvas.
 
-Additional motivations may be added to the Annotation to further clarify the intent, drawn from [extensions][ldce] or other sources. Clients _MUST_ ignore motivation values that they do not understand. Other motivation values given in the Web Annotation specification _SHOULD_ be used where appropriate, and examples are given in the [Presentation API Cookbook][cookbook].
+Additional motivations may be added to the Annotation to further clarify the intent, drawn from [extensions][prezi3-ldce] or other sources. Clients _MUST_ ignore motivation values that they do not understand. Other motivation values given in the Web Annotation specification _SHOULD_ be used where appropriate, and examples are given in the [Presentation API Cookbook][cookbook].
```
Comment (azaroth42) [inline source/api/presentation/3.0/index.md]: annex-cookbook to go with annex-services (and annex-registry?)

File: source/api/presentation/3.0/index.md [hunk 6]
```diff
@@ -957,7 +957,7 @@ Clients _SHOULD_ allow only `a`, `b`, `br`, `i`, `img`, `p`, `small`, `span`, `s
 
 The top level resource in the response _MUST_ have the `@context` property, and it _SHOULD_ appear as the very first key/value pair of the JSON representation. This tells Linked Data processors how to interpret the document. The IIIF Presentation API context, below, _MUST_ occur once per response in the top-most resource, and thus _MUST NOT_ appear within embedded resources. For example, when embedding a Canvas within a Manifest, the Canvas will not have the `@context` property.
 
-The value of the `@context` property _MUST_ be an array, and the __last__ two values _MUST_ be the Web Annotation context and the Presentation API context, in that order. Further contexts, such as those for local or [registered extensions][registry-home], _MUST_ be added at the beginning of the array.
+The value of the `@context` property _MUST_ be an array, and the __last__ two values _MUST_ be the Web Annotation context and the Presentation API context, in that order. Further contexts, such as those for local or [registered extensions][registry], _MUST_ be added at the beginning of the array.
```
Comment (azaroth42) [inline source/api/presentation/3.0/index.md]: annex-registry

File: source/api/presentation/3.0/index.md [hunk 7]
```diff
@@ -968,13 +968,13 @@ The value of the `@context` property _MUST_ be an array, and the __last__ two va
...
+Any additional properties beyond those defined in this specification or the Web Annotation Data Model _SHOULD_ be mapped to RDF predicates using further context documents. These extensions _SHOULD_ be added to the top level `@context` property, and _MUST_ be added before the above contexts. The JSON-LD 1.1 functionality of predicate specific context definitions, known as [scoped contexts][org-w3c-json-ld-scoped-contexts], _MUST_ be used to minimize cross-extension collisions. Extensions intended for community use _SHOULD_ be [registered in the extensions registry][registry], but registration is not mandatory.
```
Comment (azaroth42) [inline source/api/presentation/3.0/index.md]: annex-registry

File: source/api/search/1.0/index.md [hunk 8]
```diff
@@ -725,15 +725,15 @@ Many thanks to the members of the [IIIF][iiif-community] for their continuous en
 [semver]: http://semver.org/spec/v2.0.0.html "Semantic Versioning 2.0.0"
 [iiif-community]: {{page.webprefix}}/community/ "IIIF Community"
 [stable-version]: {{ site.url }}{{ site.baseurl }}/api/search/{{ site.search_api.latest.major }}.{{ site.search_api.latest.minor }}/ "Stable Version"
-[paging]: {{ site.url }}{{ site.baseurl }}/api/presentation/{{ site.presentation_api.latest.major }}.{{ site.image_api.latest.minor }}/
+[paging]: {{ site.url }}{{ site.baseurl }}/api/presentation/{{ site.presentation_api.stable.major }}.{{ site.image_api.latest.minor }}/
```
Comment (azaroth42) [inline source/api/search/1.0/index.md]: I don't think this will resolve after 3.0

**Pull request merged**